src/cli/version/version.go

```diff
@@ -0,0 +1,3 @@
+package version
+
+var Version = "dev"
```

Comment:
Moving to separate package to avoid cyclic imports in events.

Reply:
Let's move this out of `/cli` and into `/internal`, please.  Doesn't make sense for an API layer to provide configuration to internal packages.

Reply:
👍 . @ryanfkeepers - if rest of the changes look good to you - let's approve so that @meain can merge once he's done without another review cycle.